tests/gd2/php_bug_72339.c

```diff
@@ -8,14 +8,26 @@ int main()
 {
 	gdImagePtr im;
 	FILE *fp;
-	
-	fp = gdTestFileOpen2("gd2", "php_bug_72339_exp.gd2");
-	im = gdImageCreateFromGd2(fp);
-	if (im == NULL) {
-		return 0;
-	} else {
-		gdTestErrorMsg("Image should have failed to be loaded\n");
-		return 1;
+	long i;
+
+	/* we're creating the test image dynamically, due to it's size */
+	fp = gdTestTempFp();
+	fputc(0x00, fp); fputc(0x02, fp);
```

Comment:
can't you declare a char[] array for all these bytes and then fwrite them in one go rather than calling fputc many times ?